app/components/ErrorFallback/index.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Use this where required in React 16
+ */
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import messages from './messages';
+
+function ErrorFallback() {
+  return <FormattedMessage {...messages.error} />;
```

Comment:
The message key should be a param ?

Reply:
right now there is a `defaultMessage` assigned here https://github.com/react-boilerplate/react-boilerplate/pull/1969/files#diff-329cc2749c47df1ee7d353ddde8039b8R11